aggregation-worker/src/main/java/au/org/emii/aggregationworker/AggregationRunner.java

```diff
@@ -178,6 +188,8 @@ public void run(String... args) {
             }
 
             CalendarDateRange subsetTimeRange = subsetParams.getTimeRange();
+            //  Use the supplied time extent
+            subsetTimeRange = subsetParams.getTimeRange();
```

Comment:
? isn't this the same as the previous line?

Reply:
yep - removed